app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomAdapter.kt

```diff
@@ -58,8 +57,8 @@ class ChatRoomAdapter(private val serverUrl: String) : RecyclerView.Adapter<Chat
 
         fun bind(message: MessageViewModel) = with(itemView) {
             bindUserAvatar(message, image_avatar, image_unknown_avatar)
-            text_user_name.text = message.sender
-            text_message_time.text = message.time
+            text_user_name.textContent = message.sender
+            text_message_time.textContent = message.time
```

Comment:
`textContent` is not suitable here as it expects it expects a String and will coerce to this type clearing all spannables we set.

Reply:
We should change `textContent` to work with `CharSequence` instead of `String`